lib/dialects/abstract/query-generator/helpers/quote.js

```diff
@@ -48,6 +48,10 @@ function quoteIdentifier(dialect, identifier, options) {
       return Utils.addTicks(Utils.removeTicks(identifier, '`'), '`');
 
     case 'postgres':
+      if (identifier.indexOf('#>>') > 0) {
```

Comment:
quote helper should not handle this, query generator is more suited for this

Reply:
Thank you for telling me this. I'll git it a try.